ddtrace/bootstrap/sitecustomize.py

```diff
@@ -111,7 +133,17 @@ def update_patched_modules():
         update_patched_modules()
         from ddtrace import patch_all
 
+        # We need to clean up after we have imported everything we need from
+        # ddtrace, but before we register the patch-on-import hooks for the
+        # integrations.
```

Comment:
👍 . The why is because if we register a hook for a module that is already imported, then we patch the module straight-away. So if we unload it _after_ we register the hooks, we effectively remove the patching, thus breaking the tracer integration.